presentational/fragments/Container.js

```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components'
 import screenSizes from '../../utils/screen-sizes'
 
 const Container = styled.div`
-  width: 80%;
+  width: 40%;
```

Comment:
Does this impact other parts of the page? Are we not going to use a wide container? Could it make more sense to provide two versions of the container: one wide and one slim? Or have a property like 
`<Container wide={true|false} />`

Reply:
Yes this is valid